server.js

```diff
@@ -219,8 +219,19 @@ if (useAutoStoreData === 'true') {
 }
 
 // Clear all data in session if you open /prototype-admin/clear-data
+// Reset all data in session with what's in the sessionDataDefaults file.
 app.post('/prototype-admin/clear-data', function (req, res) {
   req.session.data = {}
+  const sessionDataDefaultsFile = path.join(__dirname, '/app/data/session-data-defaults.js')
+  let sessionDataDefaults = {}
+
+  try {
+    sessionDataDefaults = require(sessionDataDefaultsFile)
+  } catch (e) {
+    console.error('Could not load the session data defaults from app/data/session-data-defaults.js. Might be a syntax error?')
+  }
+
+  req.session.data = Object.assign({}, sessionDataDefaults, req.session.data) // load default data from file and insert into session.
```

Comment:
This merges `sessionDefaults` with `req.session.data`, except we know `req.session.data` will be an empty object (because we set it to `{}` on line 224) so there's nothing to merge – I think we can just set `req.session.data` to `sessionDataDefaults`?

```suggestion
  req.session.data = sessionDataDefaults
```